examples/community/package.json

```diff
@@ -5,9 +5,10 @@
   "version": "0.0.0",
   "browserslist": "> 0.5%, last 2 versions, not dead, not ios_saf < 13",
   "scripts": {
-    "dev": "parcel index.html --https --no-cache --open",
+    "dev": "vite",
     "prebuild": "rm -rf dist",
-    "build": "parcel build index.html --no-cache"
+    "build": "vite build",
+    "preview": "vite preview"
```

Comment:
I usually name this `start`.

Reply:
I copy-pasted `preview` from https://github.com/status-im/status-web/pull/290.

Reply:
I can change it now if blocking, np.